content/docs/inventory/stock_takes.md

```diff
@@ -57,62 +55,132 @@ Let's start a new stocktake. To do so, tap on the `New Stocktake` button in the
+
+![Stocktake add item window](/docs/inventory/images/stocktake_add_item_modal.png)
+
+Once you select an item, stocktake rows will be generated for any batches of that item with stock, and you can carry out your stocktake data entry as above.
+
+![Stocktake add item with batches](/docs/inventory/images/stocktake_newitem_with_batches.png)
+
+If there are no batches with stock for that item, your batch list will be blank. The `Add batch (+)` button will add a new blank row, where you can enter batch information and the counted number of packs.
+
+![Stocktake add item no batches](/docs/inventory/images/stocktake_newitem_no_batches.png)
+
+## Printing Stocktake sheet
 
 When viewing a specific stocktake, simply click the `Print` button which is on the top right of the page.
 When printing, a PDF file is generated for you, which will then open in a new browser tab. This can then be printed using your browser by clicking print or using `control`+`P` (if using windows) or `cmd`+`P` keys on your keyboard (if using a mac).
 
 ![Print button](/docs/introduction/images/print_button.png)
 
-This will either
+This will either:
 
-- Show a menu of possible reports for you to select from before creating a PDF. This will happen if there are more than one report defined for the `Stocktake` report type.
-- Create a PDF immediately, if there is only one report to select from
+- Create a PDF immediately, if there is only one `Stocktake` report configured
+- Show a menu of possible reports for you to select from before creating a PDF. This will happen if there is more than one report defined for the `Stocktake` report type.
 
-![Print menu](/docs/distribution/images/os_print_menu.png)
+![Print menu](/docs/inventory/images/stocktake_print_menu.png)
 
-### Filtering lines
+## Filtering lines
 
 The list of stocktake lines can get very long if you have a large stocktake. To make working with the stocktake easier to manage, you can filter the list by item name or code.
 
-Simply enter some or all of an item code in the filter input:
+In the `Filter items` search field, simply enter some or all of an item code:
 
 ![Filter by code](/docs/inventory/images/stocktake_filter_code.png)
 
-or, enter some of an item's name:
+Or, enter some of an item's name:
 
 ![Filter by code](/docs/inventory/images/stocktake_filter_name.png)
 
 You can also group the lines by item by enabling the `Group by item` switch.
+
+## Bulk actions
+
+Sometimes you may want to make changes to many or all of the lines in your stocktake. Bulk actions are available for some of these changes.
+
+### Change location
+
+Use the checkbox column to select the lines you wish to change the location for. Then select the `Change location of lines` option from the `Actions` dropdown.
+
+![Stocktake actions](/docs/inventory/images/stocktake_actions.png)
+
+This will bring up a window where you can select which location you would like to change these stock to:
```

Comment:
```suggestion
This will bring up a window where you can select which location you would like to move the stock lines to:
```

toying with the idea of using stock lines even if we don't use it anywhere else...